sources/packages/api/src/route-controllers/application/application.system.controller.ts

```diff
@@ -55,6 +54,38 @@ export class ApplicationSystemController {
     return {
       data: application.data,
       programYear: application.programYear.programYear,
+      offering: {
+        id: application.offering.id,
+        studyStartDate: application.offering.studyStartDate,
+        studyEndDate: application.offering.studyEndDate,
+        breakStartDate: application.offering.breakStartDate,
+        breakEndDate: application.offering.breakEndDate,
+        actualTuitionCosts: application.offering.actualTuitionCosts,
+        programRelatedCosts: application.offering.programRelatedCosts,
+        mandatoryFees: application.offering.mandatoryFees,
+        exceptionalExpenses: application.offering.exceptionalExpenses,
+        tuitionRemittanceRequestedAmount:
+          application.offering.tuitionRemittanceRequestedAmount,
+        offeringDelivered: application.offering.offeringDelivered,
+        offeringIntensity: application.offering.offeringIntensity,
+      },
+      program: {
+        /**
+         * for now - if credential type is `other`, then the other is send to camunda
```

Comment:
I was not able to follow the comment here. Can you elaborate more on it?

Reply:
In the program, when `credential_type` is select as `other`, users have the option to enter a free text credential_type. 
![image](https://user-images.githubusercontent.com/77353155/133508844-0b8c4116-24ec-478a-a8bd-0fe971a5936e.png)
as of now, we are not sending the free text `credential_type` in the response. In the future, if there is a required, will need to update accordingly

Reply:
perhaps say something like:

if credential type is `other`, then the string 'other' is sent to camunda. we are not sending the free text credential type when 'other' is selected

Reply:
IMO template literals(`other`) can be avoided in the comment.

Reply:
@ann-aot, I do not see a reason to have this comment at all.
I would recommend removing it.